wp-graphql-smart-cache.php

```diff
@@ -7,7 +7,7 @@
  * Author: WPGraphQL
  * Author URI: http://www.wpgraphql.com
  * Requires at least: 5.6
- * Tested up to: 6.1
+ * Tested up to: 6.4.1
```

Comment:
Latest WordPress release is 6.4.2
```suggestion
 * Tested up to: 6.4.2
```